corecito.py

```diff
@@ -163,6 +176,12 @@ def setupLogger(log_filename):
   logger.setLevel('DEBUG')
   return logger
 
+def telegram_bot_sendtext(bot_token, bot_chatID, bot_message):
+  send_text = 'https://api.telegram.org/bot' + bot_token + '/sendMessage?chat_id=' + str(bot_chatID) + '&parse_mode=Markdown&text=' + bot_message
+  response = requests.get(send_text)
+
+  return response.json()
```

Comment:
I suggest creating a Telegram class in another file so we split functionalities, so:
- telegram.py
class Telegram:
    this commented code here

- corecito.py
from telegram import Telegram
...
Telegram.telegram_bot_sendtext(bot_token, bot_chatID, bot_message)

At this point, you can rename the function name as the class name implies the reference to telegram:
Telegram.sendtext(bot_token, bot_chatID, bot_message)

Reply:
If implemented, should bot_token and bot_chatID be included in the class telegram? or here? 
If implemented in telegram.py code would be cleaner like:
Telegram.sendtext(bot_message) 
:D

Reply:
Yes, great idea. Sure, the telegram class when initialized, it will set the token and chat_id so the call to send notification will only have the message as input parameter as you mentioned 👍